site/content/contribute/server/developer-setup/arch.md

```diff
@@ -21,41 +21,49 @@
     ```
     If you don't migrate your data, the new, docker-compose-managed containers will start out empty. To remove the old containers -- destroying any existing data -- use `make clean-old-docker`.
     
-3. Install Go:
+2. Install docker-compose
+
+    ```sh
+    sudo curl -L "https://github.com/docker/compose/releases/latest/download/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+    sudo chmod +x /usr/local/bin/docker-compose
+    ```
```

Comment:
Instead of using this approach you can install it directly via `pacman`:
`sudo pacman -S docker-compose`

Reply:
Ah yes, you're right. The reason that docker-compose is usually out of date in distro repos of course doesn't apply to Arch.

Fixed it in a new commit.

Reply:
Thanks :+1: